plugin-desktopswitch/desktopswitch.cpp

```diff
@@ -132,6 +133,17 @@ void DesktopSwitch::onWindowChanged(WId id, NET::Properties properties, NET::Pro
     }
 }
 
+void DesktopSwitch::onWindowRemoved(WId id)
+{
+    const QList<QAbstractButton*> btns = m_buttons->buttons();
+    const int btn_count = btns.count();
+    for (int i = 0; i < btn_count; ++i)
```

Comment:
Just a nitpick. Is there a reason for not using C++>11 range loop?

Reply:
@palinek thank you for reviewing the code! The reason is simply that I seldomly write code in C++ so my skills are pretty basic in this language and I didn't know about the new range loop functionality.